lib/matplotlib/cbook.py

```diff
@@ -2234,6 +2234,22 @@ def _picklable_class_constructor(mixin_class, fmt, attr_name, base_class):
     return cls.__new__(cls)
 
 
+def _is_torch_array(x):
+    """Check if 'x' is a PyTorch Tensor."""
+    try:
```

Comment:
```suggestion
    # we're intentionally not attempting to import torch. If somebody
    # has created a torch array, torch should already be in sys.modules
    try:
```

let's document this for our future selves. Please also add a similar one to _ is_ jax_array.